lib/segment/src/segment/entry.rs

```diff
@@ -491,14 +491,22 @@ impl SegmentEntry for Segment {
 
         let vectors_size_bytes = total_average_vectors_size_bytes * num_points;
 
+        // Unwrap and default to 0 here because the RocksDB storage is the only faillible one, and we will remove it eventually.
+        let payloads_size_bytes = self
```

Comment:
the alternative is to change the entire call stack to the telemetry endpoint to use `CollectionResult` which is very noisy IMHO